### PR TITLE
CLOUDP-193346: Add command "atlas backups compliancepolicy enable" L1 support

### PR DESCRIPTION
**Pull request by VincentPikand**
## Proposed changes

Implemented `atlas backups compliancepolicy enable`
A confirmation prompt will be implemented with L3.
Implemented a store under `atlas`, which is strictly v2. Did not remove old store yet because some of the commands are still using it, but created a separate ticket to merge them later.

### Output

without `--watch`
```
$ atlas backups compliancepolicy enable --projectId 64db6fc73b96124b0e828a60 --authorizedEmail firstname.lastname@example.com
Backup Compliance Policy is being enabled without any configuration. Run "atlas backups compliancepolicy --help" for configuration options.
```

with `--watch`
```
$ atlas backups compliancepolicy enable --projectId 64db6fc73b96124b0e828a60 --authorizedEmail --watch firstname.lastname@example.com
Backup Compliance Policy enabled without any configuration. Run "atlas backups compliancepolicy --help" for configuration options.
```
_Jira ticket:_ CLOUDP-193346

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/backup/compliancepolicy/constants.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compliancepolicy
+
+const (
+	active = "ACTIVE"
```
[the remark this answers is not in the document]
Comment (VincentPikand) [reply internal/cli/atlas/backup/compliancepolicy/constants.go:18]: What do you mean by flyby? The `constants` file?

File: internal/cli/atlas/backup/compliancepolicy/enable.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		return err
+	}
+}
+
+func (opts *EnableOpts) enableWatcher() (bool, error) {
+	res, err := opts.store.DescribeCompliancePolicy(opts.ConfigProjectID())
+	opts.policy = res
+	if err != nil {
+		return false, err
+	}
```
Comment (matt-condon) [inline internal/cli/atlas/backup/compliancepolicy/enable.go:58]:
```suggestion
	if err != nil {
		return false, err
	}
	opts.policy = res
```

File: internal/cli/atlas/backup/compliancepolicy/constants.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (matt-condon) [reply internal/cli/atlas/backup/compliancepolicy/constants.go:18]: Yep

File: internal/cli/atlas/backup/compliancepolicy/enable.go (hunk 2 above)
Comment (VincentPikand) [reply internal/cli/atlas/backup/compliancepolicy/enable.go]: Thanks for the spot, this was also in `setup.go` so fixed there as well.

File: internal/cli/atlas/backup/compliancepolicy/constants.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (VincentPikand) [reply internal/cli/atlas/backup/compliancepolicy/constants.go]: removed the file and moved the constant.

**Review by matt-condon — APPROVED** [review]
LGTM

**Pull request merged**